server/app.js

```diff
@@ -31,6 +33,8 @@ app.use("/", indexRouter);
 app.use("/login", validateEntryReq, loginRouter);
 app.use("/ping", pingRouter);
 app.use("/register", validateEntryReq, registerRouter);
+app.use("/universities", verifyAuth, universityRouter);
```

Comment:
verifyAuth middleware should be in routes. The signup page needs to get a list of universities without verification. Generally, it is better to put authorization in the route section to separate the public routes from the private ones.